src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs

```diff
@@ -268,6 +272,66 @@ public async Task<int> GenerateAIReview(string reviewId, string revisionId)
             return result.Violations.Count;
         }
 
+        /// <summary>
+        /// Logic to update Reviews in a blackground task
+        /// </summary>
+        /// <param name="updateDisabledLanguages"></param>
+        /// <param name="backgroundBatchProcessCount"></param>
+        /// <returns></returns>
+        public async Task UpdateReviewsInBackground(HashSet<string> updateDisabledLanguages, int backgroundBatchProcessCount)
+        {
+            foreach (var language in LanguageService.SupportedLanguages)
+            {
+                if (updateDisabledLanguages.Contains(language))
+                {
+                    _telemetryClient.TrackTrace("Background task to update API review at startup is disabled for langauge " + language);
+                    continue;
+                }
+                var languageService = LanguageServiceHelpers.GetLanguageService(language, _languageServices);
+                if (languageService == null)
+                    return;
```

Comment:
```suggestion
                    continue;
```

Reply:
Sure about this? https://github.com/Azure/azure-sdk-tools/blob/8b4943ac480d9cd3c699507d8cfb1221cd9c8a01/src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs#L361

Reply:
Yes, It's a bug in current code if we are returning.